homeassistant/components/volvooncall/lock.py

```diff
@@ -31,15 +32,30 @@ class VolvoLock(VolvoEntity, LockEntity):
 
     instrument: Lock
 
-    @property
-    def is_locked(self) -> bool | None:
-        """Return true if lock is locked."""
-        return self.instrument.is_locked
+    def __init__(
+        self,
+        coordinator: VolvoUpdateCoordinator,
+        vin: str,
+        component: str,
+        attribute: str,
+        slug_attr: str,
+    ) -> None:
+        """Initialize the lock."""
+        super().__init__(vin, component, attribute, slug_attr, coordinator)
+        self._attr_is_locked = self.instrument.is_locked
+
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        """Handle updated data from the coordinator."""
+        self._attr_is_locked = self.instrument.is_locked
```

Comment:
Same approach as for the binary sensor: Implement the `is_locked` property instead and remove `_handle_coordinator_update`.